installation/resources/production/ory-persistance-profile.yaml.tpl

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ory-overrides
+  namespace: kyma-installer
+  labels:
+    installer: overrides
+    component: ory
+    kyma-project.io/installation: ""
+data:
+  postgresql.enabled: "true"
+  hydra.hydra.autoMigrate: "true"
+  global.ory.hydra.persitance.enabled: "true"
+  # global.ory.hydra.persitance.user: "someUser"
```

Comment:
it looks the same as commented values in `values.yaml` chart. We have separate PR with docs that should describe and override for external DB.